Automattic/wp-calypso
# Copy Site: Update design for Copying Progress step

#### Proposed Changes

Redesign the Copying Progress step:
* Accept props in `ProcessingStep`
* Pass Title and Subtitle from the step builder
 
I didn't add the `< Back to sites` button.

#### Testing Instructions



* Go to `/sites`
* Click on the action menu on a business site
* Click on Copy Site
* After purchasing the plan, Observe the new Processing copy screen.

#### Screenshots



<img width="1728" alt="Screenshot 2023-01-18 at 20 43 30" src="https://user-images.githubusercontent.com/779993/213292377-83849cab-6503-4ef4-8509-54459c063e3a.png">



#### Pre-merge Checklist





- Closes https://github.com/Automattic/dotcom-forge/issues/1480
